package-drone/src/main/java/de/dentrassi/pm/jenkins/Uploader.java

```diff
@@ -13,12 +13,32 @@
 import java.io.Closeable;
 import java.io.File;
 import java.io.IOException;
+import java.util.Map;
 
 public interface Uploader extends Closeable
 {
-    public void upload ( File file, String filename ) throws IOException;
+    /**
+     * Add a single file for upload
+     *
+     * @param file
+     * @param filename
```

Comment:
No description for @param file, expected something like `the artifact name on the server`